.github/workflows/ci.yml

```
@@ -0,0 +1,208 @@
+        # Crear estructura de carpetas
+        mkdir -p deploy/frontend deploy/backend
+        
+        # Copiar archivos compilados
+        cp -r frontend/build/* deploy/frontend/
+        cp -r backend/dist/ deploy/backend/
+        cp backend/package*.json deploy/backend/
+        cp -r backend/prisma/ deploy/backend/
+
+    # Paso 6: Guardar paquete para el siguiente job
+    - name: 💾 Save deployment package
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)